cmd/root.go

```diff
@@ -291,9 +296,8 @@ var detectFilesCmd = &cobra.Command{
 			fmt.Println("Error Parsing Output:", err)
 			os.Exit(1)
 		}
-		retCode := apiInstance.GetReturnCode()
-		klog.V(5).Infof("retCode: %d", retCode)
-		os.Exit(retCode)
+		exitCode := apiInstance.GetReturnCode()
+		klog.V(5).Infof("retCode: %d", exitCode)
```

Comment:
Nit: can we change the log message to something more descriptive, like "setting exit code"